handlers/people-handler/multiple.py

```diff
@@ -0,0 +1,191 @@
+# The file will generate a rendering if the image contains more than 3 people
+
+import supplementary as s
+import logging
+logging.basicConfig(level=logging.NOTSET)
+
+# get the dominant emotion.
+# If multiple people have different emotions then render the most common one
+
+
+def calculate_dominant_emotion(happy, sad, neutral):
+    if (happy >= 1 and neutral >= 1 and sad == 0):
+        return " mostly having a happy or neutral expression "
+    if (sad >= 1 and neutral >= 1 and happy == 0):
+        return "mostly having a sad or neutral expression "
+    if (happy > 0 and neutral == 0 and sad == 0):
+        return "mostly being happy "
+    if (sad > 0 and neutral == 0 and happy == 0):
+        return "mostly being sad "
+    if (neutral > 0 and sad == 0 and happy == 0):
+        return "mostly having a neutral expression "
+    if (happy >= 1 and sad >= 1 and neutral == 0):
```

Comment:
This would mean that if there are many sad, and one happy, that it would be marked mostly happy? But only if there are zero neutral? It is very difficult to parse these, and I'm not sure they will give results that make sense.